fhir-resource-tooling/src/main/java/com/ibm/cohort/tooling/fhir/ValueSetImporter.java

```diff
@@ -44,9 +47,31 @@
 
 		@Parameter(names = {"-h", "--help"}, description = "Show this help", help = true)
 		boolean isDisplayHelp;
+		
+		@Parameter(names = {"-p", "--file-system-output-path"}, description = "If specified, value sets will be converted to FHIR format and writen out to the file system path specified by this parameter. Value sets will not be imported into the FHIR server if this option is set.", required = false)
+		String fileSystemOutputPath;
+		
+		@Parameter(names = {"-o", "--file-system-output-format"}, description = "Format to use when exporting value sets to the file system when using the -p/--file-system-output-path parameters. Valid values are JSON or XML. If not specified, the default output format will be JSON", required = false)
+		String filesystemOutputFormat = "json";
 
 		@Parameter(description = "The list of value set set spreadsheets to import", required = true)
 		List<String> spreadsheets;
+		
+		public void validate() {
+			//check to make sure we are either exporting to file OR importing to a fhir server, but not both
+			if(fileSystemOutputPath != null && measureServerConfigFile != null) {
+				throw new IllegalArgumentException("Parameters [-m, --measure-server] and [-p, --file-system-output-path] cannot both be specified on the same invocation");
+			}
```

Comment:
I think there is a world where running with both filesystem output and upload to FHIR server enabled makes sense. For example, if the valueset upload to FHIR was failing, I might want to be able to see the file that was being produced to check its contents after the failure occurred. The MeasureImporter does this by default. It always writes out the FHIR resources that it creates so that I can review the request and response bundles.

Reply:
I agree, but I think that would be an additional enhancement to the ValueSetImporter tool that was not in the scope of this work item. If we deem that enhancement necessary, I would agree that this would be changed as part of that.